src/rprblender/engine/viewport_engine.py

```diff
@@ -586,8 +586,10 @@ def sync_update(self, context, depsgraph):
             for mat in volume_domain_mat:
                 updates.append((mat, None, None))
 
-        if context.selected_objects != self.selected_objects and not updates:
-            # only a selection change
+        # only a selection change
+        if context.selected_objects != self.selected_objects \
+                and len(updates) == 1 and isinstance(updates[0][0], bpy.types.Scene)\
+                and updates[0][1] == updates[0][2] == False:
```

Comment:
Do you mean here: `updates[0][1] == False and updates[0][2] == False` ?

Reply:
Sure, thanks.